lsp-test/lsp-test.cabal

```diff
@@ -57,7 +57,7 @@ library
                      , text
                      , transformers
                      , unordered-containers
-                     , some
+                     , some == 1.0.1
```

Comment:
why did you have to add this constraint?

Reply:
I didn't, that was me trying things in case it helped the CI. I'll kill that commit.